bspump/avro/loader.py

```diff
@@ -13,14 +13,9 @@
 
 
 def load_avro_schema(config):
```

Comment:
You need to add a documentation here, since the function can return None now.

Also, the return value has to be checked on respective places, e.g. to enforce the value when it is mandatory.

Reply:
@ateska enforcing Schema is a bad idea IMHO. For reason being the schema is already present in AVRO. 
So, the user will only pass the Schema file only when he wants to change the schema. It should not be enforced at all.

Reply:
We need to support these scenarios:

- AVRO schema is not provided and it is part of the message (schemaless approach)
- AVRO schema is provided and therefore is **enforced** on messages (schema approach)